README.md

```diff
@@ -133,59 +149,67 @@ Now that we have a test suite, we can:
 
 * Run it once
 
-```txt
-goss validate
+```console
+$ goss validate
 ...............
 
 Total Duration: 0.021s # <- yeah, it's that fast..
 Count: 15, Failed: 0
 ```
 
-* Edit it to use [templates](https://github.com/goss-org/goss/blob/master/docs/manual.md#templates), and run with a vars file
+* Edit it to use [templates](https://goss.rocks/gossfile/#templates), and run with a vars file
 
-```txt
+```console
 goss --vars vars.yaml validate
 ```
 
 * keep running it until the system enters a valid state or we timeout
 
-```txt
+```console
 goss validate --retry-timeout 30s --sleep 1s
 ```
 
 * serve the tests as a health endpoint
 
-```txt
-goss serve &
-curl localhost:8080/healthz
+```console
+$ goss serve &
+$ curl localhost:8080/healthz
 
 # JSON endpoint
-goss serve --format json &
-curl localhost:8080/healthz
+$ goss serve --format json &
+$ curl localhost:8080/healthz
 
 # rspecish response via content negotiation
-goss serve --format json &
-curl -H "Accept: application/vnd.goss-rspecish" localhost:8080/healthz
+$ goss serve --format json &
+$ curl -H "Accept: application/vnd.goss-rspecish" localhost:8080/healthz
 ```
 
 ### Manually editing Goss files
 
 Goss files can be manually edited to improve readability and expressiveness of tests.
 
-A [Json draft 7 schema](https://github.com/json-schema-org/json-schema-spec/blob/draft-07/schema.json) available in [docs/goss-json-schema.yaml](./docs/goss-json-schema.yaml) makes it easier to edit simple goss.yaml files in IDEs, providing usual coding assistance such as inline documentation, completion and static analysis. See [PR 793](https://github.com/goss-org/goss/pull/793) for screenshots.
+A [Json draft 7 schema](https://github.com/json-schema-org/json-schema-spec/blob/draft-07/schema.json) available
+at <https://goss.rocks/schema.yaml> makes it easier to edit simple goss.yaml files in IDEs,
```

Comment:
I don't know, I just updated the URL.
Versioning it was a bit out of scope, I think it needs its own PR to be able to discuss versioning rules, URLs...
Then it will be just a matter of renaming the file. In the meantime, given it was not versioned and only available as a github source file, I just exposed it on a clean URL.